pom.xml

```
@@ -591,7 +635,9 @@
                         <version>3.3.0</version>
                         <configuration>
                             <forkCount>1</forkCount>
-                            <argLine>-javaagent:/home/sarps/IdeaProjects/Ares2/target/ares-2.0.0-SNAPSHOT-agent.jar -Xbootclasspath/a:${user.home}/.m2/repository/org/aspectj/aspectjrt/${aspectj.version}/aspectjrt-${aspectj.version}.jar</argLine>
+                            <argLine>-javaagent:/home/sarps/IdeaProjects/Ares2/target/ares-2.0.0-Beta-1-agent.jar
```

This should be changed to actual argLine usedd in the other maven-surefire plugin, as this contains the agent from my home path. It is also a good idea to save the name, version, agent in different properties and use them to avoid these manual changes
```suggestion
                            <argLine>-javaagent:${project.build.directory}${file.separator}ares-2.0.0-Beta-1-agent.jar
```